discovery/manager_test.go

```diff
@@ -859,11 +859,15 @@ func TestTargetSetRecreatesEmptyStaticConfigs(t *testing.T) {
 }
 
 func TestIdenticalConfigurationsAreCoalesced(t *testing.T) {
-	tmpFile, err := ioutil.TempFile("", "sd")
+	tmpDir, err := ioutil.TempDir("", "prometheus-cfg-coalesced")
+	testutil.Ok(t, err)
+		t.Fatalf("error creating temporary directory: %v", err)
```

Comment:
then this needs to go and the next line too

Reply:
thanks!
Replaced err check with the testutil in the test